src/main/webapp/app/exam/shared/testExam-workingTime/testexam-working-time.component.html

```diff
@@ -0,0 +1,4 @@
+<span>
+    {{ usedWorkingTime | artemisDurationFromSeconds }}
+    <ng-container *ngIf="percentUsedWorkingTime !== 0">({{ percentUsedWorkingTime }}&thinsp;%)</ng-container>
```

Comment:
```suggestion
    <ng-container *ngIf="!!percentUsedWorkingTime">({{ percentUsedWorkingTime }}&thinsp;%)</ng-container>
```
i guess that should also work :+1: